great_expectations/expectations/core/expect_column_proportion_of_non_null_values_to_be_between.py

```diff
@@ -249,24 +261,153 @@ def schema_extra(
                 }
             )
 
+    @classmethod
+    def _get_min_max_string(cls, renderer_configuration: RendererConfiguration) -> str:
+        params = renderer_configuration.params
+
+        if not params.min_value and not params.max_value:
+            return "may have any fraction of non-null values."
+        else:
+            at_least_str = "greater than or equal to"
+            if params.strict_min:
+                at_least_str = cls._get_strict_min_string(
+                    renderer_configuration=renderer_configuration
+                )
+            at_most_str = "less than or equal to"
+            if params.strict_max:
+                at_most_str = cls._get_strict_max_string(
+                    renderer_configuration=renderer_configuration
+                )
+            if not params.min_value:
+                return f"fraction of non-null values must be {at_most_str} $max_value."
+            elif not params.max_value:
+                return f"fraction of non-null values must be {at_least_str} $min_value."
+            elif params.min_value.value != params.max_value.value:
+                return (
+                    f"fraction of non-null values must be {at_least_str} $min_value "
+                    f"and {at_most_str} $max_value."
+                )
+            else:
+                return "fraction of non-null values must be exactly $min_value."
+
     @classmethod
     @override
-    def _prescriptive_template(  # type: ignore[empty-body]  # TODO: remove this with impementation
+    def _prescriptive_template(
         cls,
         renderer_configuration: RendererConfiguration,
-    ) -> RendererConfiguration: ...
+    ) -> RendererConfiguration:
+        add_param_args: AddParamArgs = (
+            ("column", RendererValueType.STRING),
+            ("min_value", [RendererValueType.NUMBER, RendererValueType.DATETIME]),
+            ("max_value", [RendererValueType.NUMBER, RendererValueType.DATETIME]),
+            ("strict_min", RendererValueType.BOOLEAN),
+            ("strict_max", RendererValueType.BOOLEAN),
+        )
+        for name, param_type in add_param_args:
+            renderer_configuration.add_param(name=name, param_type=param_type)
+
+        template_str = cls._get_min_max_string(renderer_configuration)
+
+        if renderer_configuration.include_column_name:
+            template_str = f"$column {template_str}"
+
+        renderer_configuration.template_str = template_str
+
+        return renderer_configuration
 
     @classmethod
     @override
     @renderer(renderer_type=LegacyRendererType.PRESCRIPTIVE)
     @render_suite_parameter_string
-    def _prescriptive_renderer(  # type: ignore[empty-body]  # TODO: remove this with impementation
+    def _prescriptive_renderer(
```

Comment:
I was unable to find any existing testing patterns for these legacy renderers. I can unit test it, but I'm not sure how that adds much value. I will test this manually with Core Data Docs.